modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -42,8 +42,10 @@ interface ExtractColsDetail {
 
 type CompElm = (e1: SugarElement, e2: SugarElement) => boolean;
 
+// This uses a slight variation to the default `ContentEditable.isEditable` behaviour,
+// as when the element is detached we assume it is editable instead as it'll be a new cell.
```

Comment:
Just a slight tweak to avoid the two 'as' in a row - up to you

```suggestion
// This uses a slight variation to the default `ContentEditable.isEditable` behaviour,
// as when the element is detached we assume it is editable because it is a new cell.
```